test/regression.uts

```diff
@@ -3827,933 +1892,6 @@ p = Ether()/IPv6(dst="www.google.com")/TCP()
 assert p.dst != p[IPv6].dst
 p.show()
```

Comment:
Is there a reason you left this one?

Reply:
I was thinking to move it to l2.uts as the test name suggests.

Reply:
Makes sense 👍